README.md

```diff
@@ -562,6 +563,29 @@ To use Amazon S3 as a source of input files, use AWS credentials as instructed b
 
 * --awsCredentialsProvider: The AWS credentials provider.
 
+### GCS as sink for DEID
```

Comment:
Is it possible to mention this in 'Table of Contents'?

Reply:
Made changes